### PR TITLE
Subnumbering the chemical symbols.

### PR DESCRIPTION
**Pull request by Ethryan**
Warning, it looks awful in the code, but it works up to the steel number Fe50C.
image
I tried for a long time to make the code more compressed but was unsuccessful.

### CODE REVIEW

**Comment by Dream-Master** [comment]
@Technus please check code. Thanks

**Comment by Technus** [comment]
I nearly did KMS with this.

File: src/main/java/gregtech/api/objects/MaterialStack.java [hunk 1]
```diff
@@ -31,16 +31,402 @@ public boolean equals(Object aObject) {
...
+         }
+		 if (mAmount == 50) {
+             temp4 = "\u2085"+"\u2080";
+             
+             if (mMaterial.mMaterialList.size() > 1 || isMaterialListComplex(this)) {
+                temp1 = "(";
+                temp3 = ")";
+             }
+         }
+
```
Comment (Technus) [inline src/main/java/gregtech/api/objects/MaterialStack.java]: NO

**Comment by Ethryan** [comment]
Sorry Technus, i'm not a good coder.

File: src/main/java/gregtech/api/objects/MaterialStack.java (hunk 1 above)
Comment (Ethryan) [reply src/main/java/gregtech/api/objects/MaterialStack.java]: Good luck i leave it in your capable hands.

**Comment by Technus** [comment]
Quasi merged.

**Pull request merged**